repositories/loaders.json

```diff
@@ -0,0 +1,45 @@
+[
```

Comment:
Separation is well received, but we can also separate them using an object. What do you think?

Reply:
That would make sense as well. I'd rather spend my time doing whatever needs to be done with the order of the steps. Where should proselint be run?

Reply:
Before building and deploying